zone/client_packet.cpp

```diff
@@ -10859,7 +10859,50 @@ void Client::Handle_OP_MoveItem(const EQApplicationPacket *app)
 
 void Client::Handle_OP_MoveMultipleItems(const EQApplicationPacket *app)
 {
-	Kick("Unimplemented move multiple items"); // TODO: lets not desync though
+	if (m_ClientVersionBit & EQ::versions::maskRoF2AndLater) {		
+		if (!CharacterID()) {
+			return;
+		}
+
+		if (app->size < sizeof(MultiMoveItem_Struct)) {
+			return; // Not enough data to be a valid packet
```

Comment:
if we return should we kick else there is a chance of desync and item loss? I assume the client is still honoring the attempt to move.

Reply:
Yes, should probably go linkdead here instead.

Reply:
HandleOP_MoveItem handles these situations by just returning, which is a similar desync situation.